### ISSUE TITLE
Upgrade MQTT to v2.5+

### ISSUE THREAD

**Issue by sjmf** [sjmf/ng2-mqtt-demo#9]
The story so far...

Referencing PR #7, this issue tracks updating to a more modern MQTT.js version for this project.

Actions:
 * remove @types/mqtt from package.json
 * Code compatibility changes (`mqtt.ClientOptions` => `mqtt.IClientOptions`)
 * Check & test connection to broker.

**Comment by sjmf** [comment]
Using a testing instance of RabbitMQ via Docker:

```
FROM rabbitmq:latest

RUN rabbitmq-plugins enable --offline rabbitmq_mqtt
RUN rabbitmq-plugins enable --offline rabbitmq_web_mqtt
RUN rabbitmq-plugins enable --offline rabbitmq_stomp
RUN rabbitmq-plugins enable --offline rabbitmq_web_stomp
RUN rabbitmq-plugins enable --offline rabbitmq_management

EXPOSE 1883 4369 5671 5672 15672 15674 15675 25672 61613
```

Dockerfile here for reference.

**Comment by sjmf** [comment]
Working on this now (currently failing to connect to my message queue instance, but suspect it's a config issue rather than a library one.)

EDIT: Yes, it wouldn't let me connect to `ws://localhost:15675/ws` , but `ws://127.0.0.1:15675/ws` was fine... * *facepalm* *

**Comment by sjmf** [comment]
Resolved in e3358dd
